js_modules/dagster-ui/packages/ui-core/src/automation/MergedAutomationRoot.tsx

```diff
@@ -114,8 +114,12 @@ export const MergedAutomationRoot = () => {
       repoBuckets,
       useCallback((repoBucket) => {
         return [
-          ...repoBucket.schedules.flatMap((schedule) => schedule.tags),
-          ...repoBucket.sensors.flatMap((sensor) => sensor.tags),
+          ...repoBucket.schedules.flatMap((schedule) =>
+            schedule.tags.map(({key, value}) => ({key, value})),
+          ),
+          ...repoBucket.sensors.flatMap((sensor) =>
+            sensor.tags.map(({key, value}) => ({key, value})),
+          ),
```

Comment:
Remove the `__typename` key from the tags so that they don't end up encoded in the URL.